htdocs/api/v0.0.3-dev/candidates/visits/images/headers/Headers.php

```diff
@@ -86,6 +86,13 @@ public function handleGET()
                        "Length"   => $this->getHeader("time:length"),
                        "StepSize" => $this->getHeader("time:step"),
                       ];
+
+        $Manufacturer    = $this->getHeader("study:manufacturer");
+        $Model           = $this->getHeader("study:device_model");
+        $SoftwareVersion = $this->getHeader("study:software_version");
+        $SerialNumber    = $this->getHeader("study:serial_no");
+        $FieldStrength   = $this->getHeader("study:field_value");
```

Comment:
```suggestion
        $manufacturer    = $this->getHeader("study:manufacturer");
        $model           = $this->getHeader("study:device_model");
        $softwareVersion = $this->getHeader("study:software_version");
        $serialNumber    = $this->getHeader("study:serial_no"); 
        $fieldStrength   = $this->getHeader("study:field_value");
```
regular variable should start with a lower case letter (compare to Object variable that should start with a capital letter

Reply:
done but only for the variables I added. Variables above in the code starts with capital letters. @driusan, @PapillonMcGill  Should I change it or it is outside of the scope of this PR? Let me know.